packages/components/psammead-figure/package-lock.json

```diff
@@ -0,0 +1,399 @@
+{
```

Comment:
The package lock needs updating with a clean `npm install`

Reply:
oh that's right thanks.